uber/utils.py

```diff
@@ -340,8 +340,8 @@ def to_dict(self):
         }
 
     @property
-    def is_empty(self):
-        return not self._targets
+    def has_targets(self):
+        return not not self._targets
```

Comment:
I don't want to return `self._targets` directly because I don't want to expose it to the calling function. `self._targets` might have a list of model objects or a list of dicts, and I don't want the calling function to fool with the internals of the `Charge` class. So I coerce it to a boolean with the `not not` construction.